src/external-scripts/index.ts

```diff
@@ -22,29 +22,24 @@ export type TrackableSession = {
 // Inject scripts and run any init code
 const injectScripts = ({
   hubspotTrackingId,
-  googleTagManagerAuthToken,
-  gtmPreview,
   announcementEnabled,
-  oneTrustDomain,
-  oneTrustEnabled,
-  oneTrustTest,
   inkeepEnabled,
   inkeepApiKey,
   inkeepIntegrationId,
   inkeepOrganizationId,
+}: {
```

Comment:
This can be `ExternalScriptsData`, there's full overlap

Reply:
ah man, I've merged them all together